target/linux/ramips/image/rt3883.mk

```diff
@@ -59,6 +59,21 @@ define Device/edimax_br-6475nd
 endef
 TARGET_DEVICES += edimax_br-6475nd
 
+define Device/engenius_esr600h
+  $(Device/uimage-lzma-loader)
+  SOC := rt3662
+  BLOCKSIZE := 4k
+  IMAGE_SIZE := 7872k
+  IMAGES += factory.dlf
+  IMAGE/factory.dlf := $$(sysupgrade_bin) | check-size | \
+	senao-header -r 0x101 -p 0x44 -t 2
+  DEVICE_VENDOR := EnGenius
+  DEVICE_MODEL := ESR600H
+  DEVICE_PACKAGES := kmod-usb-ohci kmod-usb2 uboot-envtools
+  SUPPORTED_DEVICES += esr600h
```

Comment:
Is this required for upgrade from OEM? Otherwise, it can be dropped.

Reply:
ja